### PR TITLE
Add ios_ntp_global resource module

### PR DESCRIPTION
**Pull request by KB-perByte**
##### SUMMARY

Add ios_ntp_global resource module

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

ios_ntp_global resource module

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by KB-perByte** [comment]
Module | statements | missing | excluded | branches | partial | coverage
-- | -- | -- | -- | -- | -- | --
Total | 193 | 2 | 0 | 46 | 3 | 98%
plugins/module_utils/network/ios/argspec/ntp_global/__init__.py | 0 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/ios/argspec/ntp_global/ntp_global.py | 4 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/ios/config/ntp_global/__init__.py | 0 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/ios/config/ntp_global/ntp_global.py | 63 | 0 | 0 | 32 | 1 | 99%
plugins/module_utils/network/ios/facts/ntp_global/__init__.py | 0 | 0 | 0 | 0 | 0 | 100%
plugins/module_utils/network/ios/facts/ntp_global/ntp_global.py | 35 | 1 | 0 | 12 | 1 | 96%
plugins/module_utils/network/ios/rm_templates/ntp_global.py | 8 | 0 | 0 | 0 | 0 | 100%
plugins/modules/ios_ntp_global.py | 15 | 1 | 0 | 2 | 1 | 88%
tests/unit/modules/network/ios/test_ios_ntp_global.py | 68 | 0 | 0 | 0 | 0 | 100%

File: plugins/modules/ios_ntp_global.py [hunk 1]
```diff
@@ -0,0 +1,1088 @@
...
+      authentication_keys:
+        - algorithm: md5
+          encryption: 22
+          id: 2
+          key: SomeSecurePassword
+      broadcast_delay: 22
+      clock_period: 5
+      logging: true
+      master:
+        stratum_number: 4
```
Comment (NilashishC) [inline plugins/modules/ios_ntp_global.py:313]:
```suggestion
        stratum: 4
```

File: plugins/modules/ios_ntp_global.py [hunk 2]
```diff
@@ -0,0 +1,1088 @@
...
+#         }
+#     ],
+#     "update_calendar": true
+# }
+"""
+
+RETURN = """
+before:
+  description: The configuration prior to the module execution.
+  returned: when state is I(merged), I(replaced), I(overridden), I(deleted) or I(purged)
```
Comment (NilashishC) [inline plugins/modules/ios_ntp_global.py:1011]: These have been updated in latest builder about ~20 days back. Please have a look at the template.

File: docs/cisco.ios.ios_ntp_module.rst [hunk 3]
```diff
@@ -14,6 +14,13 @@ Version added: 1.0.0
    :local:
    :depth: 1
 
+DEPRECATED
+----------
+:Removed in collection release after 
+:Why:
```
Comment (NilashishC) [inline docs/cisco.ios.ios_ntp_module.rst:20]: Did this not get populated?

Comment (KB-perByte) [reply docs/cisco.ios.ios_ntp_module.rst:20]: Yes, This is breaking Sanity I am wondering why.

**Review by NilashishC — APPROVED** [review]

**Comment by KB-perByte** [comment]
recheck

**Pull request merged**